mlir/test/python/ir/attributes.py

```diff
@@ -348,6 +348,10 @@ def testDenseIntAttr():
         # CHECK: i1
         print(ShapedType(a.type).element_type)
 
+        shape = Attribute.parse("dense<[0, 1, 2, 3]> : vector<4xindex>")
+        # CHECK: attr: dense<[0, 1, 2, 3]> : vector<4xindex>
```

Comment:
```suggestion
        # CHECK: attr: dense<[0, 1, 2, 3]>
```
CHECK lines above omit the type